GitCommandsTestWithConsoleApp/obj/Debug/GitCommandsTestWithConsoleApp.csproj.CoreCompileInputs.cache

```diff
@@ -1 +1 @@
-8da5fbcb33b595602d5f7d2c590d0554bce8e149
+044f68d0a95dd498c0690d17254eed5dc4ad4bd2
```

Comment:
Debug folder should not be included in a git repository. Add it to the .gitignore file.

Reply:
Added it to .gitignore in commit # 387e953.